torch_brain/models/ndt2.py

```
@@ -0,0 +1,526 @@
+        super().__init__()
+        self.is_ssl = is_ssl
+        self.is_causal = is_causal
+
+        self.ctx_time = ctx_time
+        self.bin_time = bin_time
+        if not np.isclose(
+            ctx_time - bin_time * np.round(ctx_time / bin_time), 0, atol=1e6
+        ):
+            raise ValueError(
+                f"ctx_time should be a multiple of bin_time (ctx_time:{ctx_time}, bin_time:{bin_time})"
+            )
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)